website/utils/url_utils.ts

```diff
@@ -2,16 +2,47 @@ import FirebaseRequest, { DatasetMetaData } from "../../public/firebase";
 
 import { AppProps, GlobalViewerSettings } from "../../src/aics-image-viewer/components/App/types";
 import { ViewMode } from "../../src/aics-image-viewer/shared/enums";
-import { ViewerChannelSettings } from "../../src/aics-image-viewer/shared/utils/viewerChannelSettings";
+import {
+  ViewerChannelSetting,
+  ViewerChannelSettings,
+} from "../../src/aics-image-viewer/shared/utils/viewerChannelSettings";
+
+const CHANNEL_STATE_KEY_REGEX = /^c[0-9]+$/;
+// Match colon-separated pairs of alphanumeric strings
+const LUT_REGEX = /^[a-z0-9.]*:[ ]*[a-z0-9.]*$/;
+const HEX_COLOR_REGEX = /^[0-9a-fA-F]{6}$/;
+
+export type ViewerChannelSettingJson = {
+  /** Color, as a 6-digit hex color */
+  c?: string;
+  /** Colorize. "1" is enabled. */
+  cz?: "1" | "0";
+  /** Colorize alpha, in the [0, 1] range. */
+  cza?: string;
+  /** Opacity, in the [0, 1 range] */
+  op?: string;
+  /** LUT to map from intensity to opacity. Should be two alphanumeric values separated
+   * by a colon, e.g. "0:255", "p50:p90", or "autoij:0".
+   */
+  lut?: string;
+  /** Volume enabled. "1" is enabled. */
+  v?: "1" | "0";
+  /** Isosurface enabled. "1" is enabled. */
+  i?: "1" | "0";
+  /** Isosurface value, in the [0, 255] range. */
+  iv?: string;
+};
```

Comment:
Anything you'd change about this naming?

Reply:
IMO, nothing super important to change - we can totally use this as is - but some ideas. 

Would it look nicer/more consistent if all the keys had the same number of characters?

Also, I believe opacity (kinda the same thing as alpha) is really only used for isosurface so it could get an `i` in front.  Also we could consider that "surface" is the key word and use `s` instead of `i`.
So you could have: `col, clz, cza, isa, lut, ven, sen (or ien), isv`: color, colorize on/off, colorize alpha, isosurface alpha, lut, volume enable, surface enable, isovalue

Reply:
Renamed! Thank you for the suggestions!